src/LibraryManager.Vsix/Json/Completion/SimpleCompletionEntry.cs

```diff
@@ -4,9 +4,9 @@
 using Microsoft.JSON.Editor.Completion;
 using Microsoft.VisualStudio.Imaging.Interop;
 using Microsoft.VisualStudio.Language.Intellisense;
-using Microsoft.Web.Editor.Completion;
```

Comment:
either revert this change or sort all of these

Reply:
Sorted.

---
In reply to: [199296051](https://github.com/aspnet/LibraryManager/pull/258#discussion_r199296051) [](ancestors = 199296051)